sklearn/svm/tests/test_svm.py

```diff
@@ -277,14 +309,13 @@ def test_probability():
 
     for clf in (svm.SVC(probability=True, random_state=0, C=1.0),
                 svm.NuSVC(probability=True, random_state=0)):
-
         clf.fit(iris.data, iris.target)
 
         prob_predict = clf.predict_proba(iris.data)
         assert_array_almost_equal(
             np.sum(prob_predict, 1), np.ones(iris.data.shape[0]))
         assert_true(np.mean(np.argmax(prob_predict, 1)
-                    == clf.predict(iris.data)) > 0.9)
+                            == clf.predict(iris.data)) > 0.9)
```

Comment:
Usually we wouldn't go fixing up cosmetic things when submitting an unrelated PR. It makes the PR somewhat harder to review. But at least this PR is small and focussed